deploy/chart/ekuiper/values.yaml

```diff
@@ -160,7 +160,7 @@ kuiperConfig:
       # to enable the checkpoint mechanism
       qos: 0
       # The interval in millisecond to run the checkpoint mechanism.
-      checkpointInterval: 300000
+      checkpointInterval: 300s
```

Comment:
Comment has to be changed. Is it possible to compatible with int, which means default to millisecond if no duration unit.

Reply:
Compatible with int will make it too complex as configuration.